### DISCUSSION TITLE
Help please! Need help using Hetty

### DISCUSSION THREAD

**Discussion by iambenkay** [dstotijn/hetty#55] [Q&A] [answered]
I have not used burp or any other web application security tool before and I seem kind of lost on how to actually use this.
Can you guide me in the right direction? I want to understand and hopefully contribute to this project in the future.

**Comment by dstotijn** [comment, marked as answer]
Hi @iambenkay, there's some rough docs [here](https://hetty.xyz/guide/modules.html#projects), but you'll probably benefit most looking into tutorials/guides/blog posts (e.g. [Hacker101](https://www.hacker101.com/), or join their Discord server to find beginner tips) that are more generic about bug bounty hunting/researching HTTP traffic. There likely be more docs/guides for Hetty in the future, but focus is on initial development first.
